### ISSUE TITLE
Dolby Vision support

### ISSUE THREAD

**Issue by satmandu** [xbmc/xbmc#22408]
With the closure of https://github.com/xbmc/xbmc/pull/18965 a new PR is needed to support Dolby Vision / DV mkv files?

(And does there need to be a separate Issue created to ask for tone mapping support for playing Dolby Vision files on Non-HDR displays?)

**Comment by Nuklear92** [comment]
I checked the latest build they released today Jan 6, build: 

**kodi-20230106-fac12d4e-Nexus-25474-arm64-v8a.apk**

I'm not sure if this build has or not the FFmpeg 5.0.x they merged a few hours ago. But, I checked a DV file I have in mkv container and still showing green tint. 😩

**Comment by fandangos** [comment]
https://github.com/xbmc/xbmc/pull/22410/

Let's see how it goes. 

But FYI there's no SDR tonemapping for Dolby Vision content **ONLY** in any device as far as I know.
Even the OPPO 203 Bluray Player uses the HDR10 metadata for SDR tonemapping. 
The same goes for Emby as far as I can test. Files with Dolby Vision layer only do not will not be tonemapped and will display a pink and purple image OR an unsatured image.

**Comment by satmandu** [comment]
Jellyfin can do tone mapping. There are flags passed to ffmpeg...

**Comment by satmandu** [comment]
It's mentioned in this thread:

https://www.reddit.com/r/jellyfin/comments/uzid62/dolby_vision_tonemapping_support_for_transcodes/iacgt9h/

**Comment by satmandu** [comment]
More specifically:

https://www.reddit.com/r/jellyfin/comments/v2nudu/dolby_vision_to_sdr_hwa_tonemapping_is_coming

**Comment by fandangos** [comment]
Well, things do change over time :)

But that tonemap is done server side, not on player side. 
Also, the Nvidia Shield now has the option to disable Dolby Vision and that forces tonemapping for the player. DV only content doesn't work.

So, as far as I know, only Jellyfin transcoding can do it. 
It might be possible to bring that to Kodi on Windows/Linux but I don't see how it can be done on Android (way above what I can do too).

**Comment by satmandu** [comment]
Agreed on tone mapping in android possibly being an issue. But for those of us running kodi standalone on a machine with the GPU capabilities to do so, it should be possible (dare I say ideal?) for kodi to do that as well, since it could also pass ffmpeg the same flags to tone map.

I'll open a separate issue for Dolby Vision tone mapping then...

**Comment by fuzzard** [comment]
Feature requests on forums as always. This is a bug tracker only
